crates/db_views/src/post_view.rs

```diff
@@ -393,6 +393,9 @@ impl<'a, 'b: 'a> PostQuery<'a, 'b> {
     query = match self.sort.unwrap_or(SortType::Hot) {
       SortType::Active => query.then_order_by(post_aggregates::hot_rank_active.desc()),
       SortType::Hot => query.then_order_by(post_aggregates::hot_rank.desc()),
+      SortType::Controversial => query
+        .then_order_by(post_aggregates::controversy_rank.desc())
+        .then_order_by(post_aggregates::hot_rank.desc()),
```

Comment:
Remove this second order by hot rank, it will probably bypass the index and be really slow.

Eventually I've got to do a 2nd order by for published on all of these, because of the weird old post bug we're seeing. But I'll do that later.

Reply:
Im not sure thats a good idea, then controversial rank would only show the most controversial posts of all time which would rarely change. Another solution might be to limit controversial sort to posts from last week?